database/index.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const mongoUri = process.env.MONGODB_URI;
+
+const db = mongoose.connect(mongoUri, {
+  useNewUrlParser: true,
```

Comment:
Great job taking care of those warnings (deprecation notices). I updated my code to use newURLParser and unifiedTopology too